lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -93,6 +99,14 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
       <%= for session <- @sessions do %>
         <div class="py-4 flex items-center border-b border-gray-300"
           data-test-session-id={session.id}>
+          <%= if @editing? do %>
+            <input
+            class="text-blue-500 mx-1 mr-3 focus:ring-blue-400 focus:ring-opacity-25 border border-gray-300 rounded-lg"
```

Comment:
I have this lying around from when we used checkbox instead of toggle in other places:

```css
.checkbox-base {
  @apply h-5 w-5 appearance-none border border-gray-300 rounded text-blue-600 cursor-pointer;
}

.checkbox-base:checked {
  background-image: url("data:image/svg+xml,%3csvg viewBox='0 0 16 16' fill='white' xmlns='http://www.w3.org/2000/svg'%3e%3cpath d='M12.207 4.793a1 1 0 010 1.414l-5 5a1 1 0 01-1.414 0l-2-2a1 1 0 011.414-1.414L6.5 9.086l4.293-4.293a1 1 0 011.414 0z'/%3e%3c/svg%3e");
  border-color: transparent;
  background-color: currentColor;
  background-size: 100% 100%;
  background-position: center;
  background-repeat: no-repeat;
}
```

You can drop it in `components.css` and see if it fits :)